script/Arcueid/Day 12/12_00_ARC12_2B.txt

```diff
@@ -329,33 +329,33 @@ Lighten up will ya, we're definitely going to kill this Dead Apostle.
 [sha:c80ae8a6672e88b559677849ec4913fba7af5680]{
 -- Page 97, Offset 30816.
 -- だが―――私は確かに書き残した筈だ。
-But I could have sworn I left behind a note―――
+Still, I'm certain I noted it down quite specifically―――
 }
 [sha:ad102f1187db5c8879f2bc85d3e3636d5db4a6f8]{
 -- Page 97, Offset 30817.
 -- 人のままでは完全な不老不死は実現しないと。
-One saying that perfect immortality is unattainable so long as you cling to being human.
+Perfect immortality is impossible to achieve while one remains human.
 }
 [sha:816a0b07efe16900be4c8d0bcc29f3b9c0c4a072]{
 -- Page 97, Offset 30818.
 -- 私の書庫を漁った盗人どもがこの忠告を受け入れなかったのなら、自業自得と言うほかない」
-If the thieves who rummaged through my library failed to heed that advice, then they're simply getting what they deserve."
+If the thieves who rummaged through my library failed to heed that advice, then they simply got their just deserts."
 }
 [sha:16a592938bcb289f3eb7e45547d0294b178d4868]{
 -- Page 98, Offset 30819.
 -- 「“……偉ぶってんじゃねえよ。ようは未完成のまま放置したってコトじゃねえか。
-"...Quit acting so arrogant. That just means you left your life's work unfinished, doesn't it?
+"Don't act so fuckin' full'a yourself. All's that mean's thatcha left it unfinished, right?
```

Comment:
"means is" or "means's" but the latter looks absolutely ridiculous

Reply:
Yeah it's overcooked